tests/generate_status_reference_data.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# Copyright (c) 2018 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import argparse
+import os
+
+import signac
+from define_status_test_project import _TestProject
+from test_project import redirect_stdout
+
+import flow
+import flow.environments
+from flow.scheduling.fake_scheduler import FakeScheduler
+
+PROJECT_NAME = "StatusTest"
+ARCHIVE_DIR = os.path.normpath(
+    os.path.join(os.path.dirname(__file__), "./status_reference_data.tar.gz")
+)
+
+
+def init(project):
```

Comment:
is it important here to check non-numeric key values, or is it handled elsewhere that you don't have to here?

Reply:
Sorry it took a while. I'm not sure what you mean. Could you explain what non-numeric key values you're referring to?

Reply:
@Charlottez112 I think @atravitz might have meant that the state points initialized in this loop only contain integer values. While it would be nice to test non-integer values (e.g. strings or floats), I don't think that is important enough to delay this PR being merged.